docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Environment Variables"
+description: Oryx environment variables
+last_updated: Sep 17, 2023
+template: concept-topic-template
+---
+
+[Environment variables](https://en.wikipedia.org/wiki/Environment_variable) - is a user-definable value that can affect the way running process(es) will behave on a computer. Environment variables are part of the environment in which a process runs.
+
+They are useful when:
+
+- Values are different across developer machines
+- Values are different across multiple environments: (dev, staging, qa, prod)
+- Values change frequently and are highly dynamic (especially when running in CI)
+
+Make sure that values **do not leak** in the code repository, especially secrets/passwords/keys etc. Keep them secure.
+
+Oryx uses vite [.env files]([https://github.com/motdotla/dotenv](https://vitejs.dev/guide/env-and-mode.html#env-files)) for environment variables management. 
+
+## Storefront Variables
+
+The list of environment variables to configure Storefront:
+
+| Variable | Required | Default | Description                               |
+|----------|----------|---------|-------------------------------------------|
+| `SCOS_BASE_URL` | Yes | None | Spryker GLUE API URL |
```

Comment:
Are we going to release this doc and then update it after 1.1 to add `ORYX_PRESET` and `ORYX_FEATURE_VERSION`? As for me we can already prepare it for release 1.1.

Reply:
Yes, we can add a note / section with 1.1 info.

Reply:
Should I add 

```
| `ORYX_PRESET` | No | `b2c` | Used to set a preset name, available options: `b2b`, `b2c` |
| `ORYX_FEATURE_VERSION` | No | `latest` | Used to set a feature level, available options: `latest`, `1.0`, `1.1`, etc. Only major and minor versions can be used |
```

back?

Reply:
Yes please, probably as a separate table with note these are availble from 1.1 only. Or you can add one more column to the table.

Reply:
2 env variables and version column was added